packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbar.md

```diff
@@ -0,0 +1,793 @@
+import React from 'react';
+import { DataToolbar, DataToolbarContent, DataToolbarGroup, DataToolbarItem } from '@patternfly/react-core/dist/esm/experimental';
+import { Button, Select, SelectOption } from '@patternfly/react-core';
+import { EditIcon, CloneIcon, SyncIcon } from '@patternfly/react-icons'
+
+class DataToolbarGroupTypes extends React.Component {
+  constructor(props) {
+    super(props);
+    
+    this.firstOptions = [
+      { value: 'Filter 1', disabled: false, isPlaceholder: true },
+      { value: 'A', disabled: false },
+      { value: 'B', disabled: false },
+      { value: 'C', disabled: false },
+    ];
+    
+    this.secondOptions = [
+      { value: 'Filter 2', disabled: false, isPlaceholder: true },
+      { value: '1', disabled: false },
+      { value: '2', disabled: false },
+      { value: '3', disabled: false },
+    ];
+    
+    this.thirdOptions = [
+      { value: 'Filter 3', disabled: false, isPlaceholder: true },
+      { value: 'I', disabled: false },
+      { value: 'II', disabled: false },
+      { value: 'III', disabled: false },
+    ];
+    
+    this.state = {
+      firstIsExpanded: false,
+      firstSelected: null,
+      secondIsExpanded: false,
+      secondSelected: null,
+      thirdIsExpanded: false,
+      thirdSelected: null,
+    };
+    
+    this.onFirstToggle = isExpanded => {
+      this.setState({
+        firstIsExpanded: isExpanded
+      });
+    };
+    
+    this.onFirstSelect = (event, selection) => {
+      this.setState({
+        firstSelected: selection,
+        firstIsExpanded: false
+      });
+    };
+    
+    this.onSecondToggle = isExpanded => {
+      this.setState({
+        secondIsExpanded: isExpanded
+      });
+    };
+    
+    this.onSecondSelect = (event, selection) => {
+      this.setState({
+        secondSelected: selection,
+        secondIsExpanded: false
+      });
+    };
+    
+    this.onThirdToggle = isExpanded => {
+      this.setState({
+        thirdIsExpanded: isExpanded
+      });
+    };
+    
+    this.onThirdSelect = (event, selection) => {
+      this.setState({
+        thirdSelected: selection,
+        thirdIsExpanded: false
+      });
+    };
+    
+  }
+  
+  render() {
+    const { firstIsExpanded, firstSelected, secondIsExpanded, secondSelected, thirdIsExpanded, thirdSelected } = this.state;
+    
+    const filterGroupItems = <React.Fragment>
+      <DataToolbarItem>
+        <Select
+          variant={SelectVariant.single}
+          aria-label="Select Input"
+          onToggle={this.onFirstToggle}
+          onSelect={this.onFirstSelect}
+          selections={firstSelected}
+          isExpanded={firstIsExpanded}
+        >
+          {this.firstOptions.map((option, index) => (
+            <SelectOption
+              isDisabled={option.disabled}
+              key={index}
+              value={option.value}
+            />
+          ))}
+        </Select>
+     </DataToolbarItem>
+     <DataToolbarItem>
+        <Select
```

Comment:
There's some inconsistent indentation/spacing here and elsewhere in this file, but I don't think we need to hold up the PR for it. I'll address cases like this when I rebase https://github.com/patternfly/patternfly-react/pull/2810 after the release, but I'm not sure if Prettier will run on the JSX in our Markdown files, so that's something else I want to look into.